oxygen.md

```diff
@@ -415,7 +415,9 @@ The mission plan should aim to pass close to these platforms, ideally multiple t
 # Real time data processing & Quality Control
 
 ## Send OceanGliders metadata and set up real time data flow 
-- to add where do users find this information
+- Similarly to other type of metadata, oxygen sensor metadata (i.e. sensor model, calibration coefficient, intermediate parameters) should be send ahead of the mission to the Data Assembly Center. There is no unique procedure fot his. Protocols, format and file naming convention should be discussed with DACs before deployment. 
```

Comment:
Even if there is not unique procedure how to exactly send oxygen data at the moment. 

Could you add the link where the user finds the information on how to send general metadata?

Reply:
and I found this newer document: https://archimer.ifremer.fr/doc/00354/46542/ 

Thierry Virginie, Bittig Henry, The Argo-Bgc Team (2021). Argo quality control manual for dissolved oxygen concentration. https://doi.org/10.13155/46542

Should we refer to this one at the moment? @vracape @cgourcuf can maybe help?